packages/type-definitions/src/runtime.ts

```diff
@@ -2,6 +2,8 @@ export default {
   rpc: {},
   types: {
     BlockWeights: 'Vec<u8>',
-    ChainId: 'u32'
+    ChainId: 'u32',
+    ScalarData: '[u8; 8]',
+    Manager: 'Vec<u8>'
```

Comment:
This should be a struct as defined in the anon node, I will try to fix this or I could help you to implement it today!

Reply:
Cool, can we merge it for now? I need to get the dApp withdrawal and deposit working today

Reply:
Ok, I will do a separate PR for it later today!